.gitignore

```diff
@@ -4,6 +4,7 @@
 .DS_Store
 .vscode
 .idea
+main.py
```

Comment:
`main.py` is part of the project codebase and should not be ignored. Please revert this change.